CMakeLists.txt

```diff
@@ -122,7 +130,6 @@ if(TRAVIS_CI_BUILD)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCC)
-  include(CheckCCompilerFlag)
   check_c_compiler_flag(-Og HAS_OG_FLAG)
 else()
   set(HAS_OG_FLAG 0)
```

Comment:
It was added in d51c0a9. I think we can remove it, it seems a misunderstanding of how check_c_compiler_flag works.